content/kubermatic/master/tutorials/13-psp/_index.en.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Pod Security Policy"
+date = 2020-04-02T12:07:15+02:00
+weight = 130
++++
+
+[Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) is a key security feature in Kubernetes. It allows cluster administrators to set [granular controls](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-reference) over security sensitive aspects of pod and container spec.
```

Comment:
Can you at least once mentioned the abbreviation (`[Pod Security Policy](...) (PSP) is bla bla bla`)?